src/fetch-wrapper.ts

```diff
@@ -41,6 +41,7 @@ export default function fetchWrapper(
     body: requestOptions.body,
     headers: requestOptions.headers as HeadersInit,
     signal: requestOptions.request?.signal,
+    redirect: requestOptions.redirect,
```

Comment:
wait where is `redirect` defined? I don't see it in `@octokit/types`?

Reply:
It's defined above into he function signature

Reply:
I'm sorry I misunderstood and didn't check 😞 I don't see where we use that option, do you know where it's coming from? I see this commit but not sure before that: https://github.com/octokit/request.js/commit/7b38319b135d6bb52dbc8f9a394856da6e6b70ce

 If anything it should be passed as `requestOptions.request.redirect`, not as `requestOptions.redirect`.

I think we shouldn't pass it through and remove the type, until someone complaints that it broke something. Sorry for the back-and-forth

Reply:
It's been defined that way for 4 years now, since the very beginning of the typescript conversion

It was first introduced in https://github.com/octokit/request.js/commit/7fe91b826a86d8d31c1d21aff9ab9180a552614d

Reply:
The option itself has been defined that way since v1.2.0
It was first introduced in https://github.com/octokit/request.js/commit/d8c60e52948a59feb8cc0d07cf8adb511ae5d672

Reply:
I'm pretty sure that was due to `node-fetch`'s behavior, you could set `redirect` to `manual` and extract the redirect URL that way. But that's not the behavior of native fetch anyway. I'd say we do not add it back, and remove `redirect` from the types